src/style.css

```diff
@@ -0,0 +1,31 @@
+html {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.main-header {
+  font-size: 17px;
+  margin: 0;
+}
+
+.preamble-header,
+.article-header {
```

Comment:
you can add common class for these elems

Reply:
add common classes